pkg/acceptance/testing.go

```diff
@@ -208,3 +210,22 @@ func ExternalProviderWithExactVersion(version string) map[string]resource.Extern
 		},
 	}
 }
+
+// SetV097CompatibleConfigPath sets a new config path in a relevant env variable for a file that is compatible with v0.97.
+func SetV097CompatibleConfigPathEnv(t *testing.T) func() {
+	t.Helper()
+	return func() {
+		home, err := os.UserHomeDir()
+		require.NoError(t, err)
+		configPath := filepath.Join(home, ".snowflake", "config_v097_compatible")
+		t.Setenv(snowflakeenvs.ConfigPath, configPath)
```

Comment:
this will set env only for the test it is invoked in, right?

Reply:
Not exactly. According to the [docs](https://pkg.go.dev/testing#T.Setenv):
> Setenv calls os.Setenv(key, value) and uses Cleanup to restore the environment variable to its original value after the test.
>
> Because Setenv affects the whole process, it cannot be used in parallel tests or tests with parallel ancestors.